stdlib/internal/SwiftSyntax/SyntaxChildren.swift

```diff
@@ -0,0 +1,28 @@
+//===------------- SyntaxChildren.swift - Syntax Child Iterator -----------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+/// A sequence that allows iteration of the children of a Syntax node.
```

Comment:
D'oh, edited my comment: I was trying to ask *why* this is not a random access collection, since it seems like the underlying abstraction would support it.  That said, I'm really just curious, no need to change it.